tests/tests.cpp

```diff
@@ -31,7 +31,7 @@ TEST(exemple, exemple_test) {
     OSRedirector oss(std::cout);
 
     try {
-        dylib lib("./dynamic_lib", dylib::extension);
+        dylib lib(dylib::working_directory, "dynamic_lib");
```

Comment:
This is not clear enough:

1. `working_directory` does say _which_ directory, and the class-namespace implies it's dylib's working directory, whatever that would be
2. I'm looking for a verb indication that the current working directory is to be used.  But... perhaps there's no decent way to achieve that.